sample_apps/run_example.sh

```diff
@@ -753,6 +755,15 @@ function edit_policy_file_OE() {
     run_popd
 }
 
+# ###########################################################################
+function build_policy_generator_OE() {
+    run_cmd
+    run_pushd "${CERT_PROTO}/utilities"
+    local mkf="policy_generator.mak"
+    LOCAL_LIB=${Local_lib_path} run_cmd make -f ${mkf}
```

Comment:
On hind-sight, this step was just plain missing in earlier versions of this script. So, it's not really a regression, but just an incomplete feature support.

I simply don't recall if the interface for `setup_with_auto_policy_generation_for_OE` ever even succeeded previously.